src/components/Card/Card.react.js

```diff
@@ -35,7 +35,7 @@ function Card({
     return (
       <Component className={classes}>
         {card_header}
-        {card_body}
+        {card_body || children}
```

Comment:
I think we should have one or the other as props... seems kinda redundant to have both